### ISSUE TITLE
Closing a tab in Brave results in the browser exiting split view mode on Mac OS X

### ISSUE THREAD

**Issue by devillex** [brave/browser-laptop#2828]
**Did you search for similar issues before submitting this one?**

Yes. I couldn't find anything under both fullscreen and split view.

**Describe the issue you encountered:**

Closing a tab when the browser is in fullscreen split view mode causes the browser to exit split view.

**Expected behavior:**

Closing a tab should not alter the split view state of the browser window.

- Platform (Win7, 8, 10? macOS? Linux distro?):
```
Mac OS X El Capitan, version 10.11.6 (15G31)
```

- Brave Version:
```
Brave: 0.11.1
Electron: 1.2.7
libchromiumcontent: 51.0.2704.103
V8: 5.1.281.65
Node.js: 6.1.0
Update Channel: dev
```

- Steps to reproduce:
    1. Open a couple of tabs in Brave
    2. Enter split view mode with Brave and any other window
    3. Close an open tab in Brave

- Screenshot if needed:
```
N/A
```

- Any related issues:
```
N/A
```

**Comment by neeklamy** [comment]
Seeing as split screen mode is a type of fullscreen, I’d guess this is fixed with the *Exit of fullscreen view when a tab is closed* #2618. If so it’ll arrive with Brave 0.11.2.

**Comment by neeklamy** [comment]
Seeing as split screen mode is a type of fullscreen, I’d guess this is fixed with af7ffb1. If so it’ll arrive with Brave 0.11.2.

**Comment by luixxiul** [comment]
Yes it will. You can see this fixed on master branch, so closing. Thanks for your report.
